elfloader-tool/src/arch-arm/armv/armv7-a/32/mmu-hyp.S

```diff
@@ -81,5 +81,7 @@ BEGIN_FUNC(arm_enable_hyp_mmu)
     mcr     p15, 0, r0, c1, c0, 1
 #endif /* CONFIG_MAX_NUM_NODES > 1 */
 
+	isb sy
```

Comment:
you need an ISB after modifying system control registers to ensure the following instructions observe the changes to system control registers.

Reply:
Will update :+1: